### ISSUE TITLE
 How to use the other pre-training objectives like masked sequence-to-sequence modeling?

### ISSUE THREAD

**Issue by zhl98** [RUCAIBox/TextBox#336]
Hello, I am very interested in your work.
I know  ”--pretrain_task=denoising“  can use denoising auto-encoding.
However, I did not see the relevant parameter descriptions and restrictions in the code.
When I want to use other pre training methods, How do I set this parameter?

Thank you very much for your work, and I hope you can tell me the method when you have time.

**Comment by StevenTang1998** [comment]
We are now testing other pre-training objectives, and we will release it soon. Thanks for your reporting!
